zstacklib/zstacklib/scripts/rbd_rw.py

```
@@ -0,0 +1,237 @@
+def _log(content):
+    sys.stderr.write(content + "\n")
+    sys.stderr.flush()
+
+listen_signal()
+if __name__ == '__main__':
+    if len(sys.argv) == 1:
+        listen_pipe()
+        sys.exit(0)
+
+    if len(sys.argv) != 5 and len(sys.argv) != 3:
+        print >> sys.stderr, "Usage: {} <operation> <pool_name>/<image_name> [offset] [size/content]".format(sys.argv[0])
```

Ruff (0.8.2)

Use of `>>` is invalid with `print` function

(F633)